src/commitment_scheme/kzg10/key.rs

```diff
@@ -161,10 +161,10 @@ impl CommitKey {
         &self,
         poly_degree: usize,
     ) -> Result<(), Error> {
-        match (poly_degree == 0, poly_degree > self.max_degree()) {
```

Comment:
We should rename the function and update the documentation

Reply:
aggree! I'll do it asap!

Reply:
done